compiler/ballerina-lang/src/main/java/io/ballerina/projects/configurations/ConfigVariable.java

```diff
@@ -48,4 +50,8 @@ public boolean isRequired() {
         return isRequired;
     }
 
+    public String getDescription() {
+        return description;
```

Comment:
Shall we rename this getter to `description` to follow the pattern used in other places??

Reply:
Please update other methods as well

Reply:
Fixed with eeca230